lightning/src/routing/router.rs

```diff
@@ -679,6 +698,7 @@ pub fn get_route<L: Deref>(our_node_pubkey: &PublicKey, network: &NetworkGraph,
 								total_fee_msat: u64::max_value(),
 								htlc_minimum_msat: $directional_info.htlc_minimum_msat,
 								path_htlc_minimum_msat,
+								next_hops_penalty_msat,
```

Comment:
I believe this default should be `u64::max_value()`, which also lets us move the `next_hops_penalty_msat` down into the `should_process` block.

It doesn't make much difference in performance, however, compared to my above numbers (though I believe this may be distorted by `should_process` being always true with test set).

```
test routing::router::benches::generate_mpp_routes        ... bench:  37,355,034 ns/iter (+/- 21,957,912)
test routing::router::benches::generate_routes            ... bench:  37,530,066 ns/iter (+/- 25,158,746)
```